.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+// See https://aka.ms/vscode-remote/devcontainer.json for format details.
+{
+	"image": "ludeeus/container:integration",
+	"context": "..",
+	"appPort": [
+		"9123:8123"
+	],
+	"postCreateCommand": "container install",
+	"runArgs": [
+		"-v",
+		"${env:HOME}${env:USERPROFILE}/.ssh:/tmp/.ssh"
+	],
+	"extensions": [
+		"ms-python.python",

```

Comment:
Could we add PyLance? -> https://devblogs.microsoft.com/python/announcing-pylance-fast-feature-rich-language-support-for-python-in-visual-studio-code/

Reply:
Sure.